integration_tests/tests/consistency/consistency_tasks.sql

```diff
@@ -0,0 +1,45 @@
+{{ config(
```

Comment:
FYI this test is failing for me on tasks with multiple tags or projects because the `string_agg` is aggregating them in different orders.  So the rows are essentially the same, but the test thinks they're different because `a, b, c` is not exactly `c, b, a`. 

I think this is fine for now/for this current ticket, but would you mind creating a FR ticket to apply predictable ordering to our `string_agg`'d columns?

Reply:
cc @fivetran-joemarkiewicz

Reply:
[Feature created](https://github.com/fivetran/dbt_asana/issues/38), feel free to add any examples of how we've best handled this in the past.

Reply:
Alternatively, you could exclude the string_agg's cols. [Example](https://github.com/fivetran/dbt_amplitude/blob/main/integration_tests/tests/consistency/consistency_daily_performance.sql#L7).

Reply:
Good catch! Added for now.